src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -183,9 +183,11 @@ def acr_login(cmd, registry_name, resource_group_name=None, username=None, passw
                        "--password", password,
                        login_server])
             p.wait()
-        elif b'--password-stdin' in stderr:
-            pass
         else:
+            if b'--password-stdin' in stderr:
```

Comment:
In case of successful login, docker client prints this message to stderr with a status code 0.
In case of unsuccessful login, docker client also prints this message to stderr with a non-zero status code.